docs/logs/complex-objects/FoodRecallNotice.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+public class FoodRecallNotice
```

Comment:
I intend to keep this code small instead of mirroring what .NET official docs will be doing.

This will be documented by .NET official docs https://github.com/dotnet/extensions/blob/7db3a7edc35ae8008c75631da35c01467912cacf/src/Libraries/Microsoft.Extensions.Telemetry.Abstractions/Logging/LogPropertiesAttribute.cs#L49.